benchmarks/src/jmh/java/com/linecorp/armeria/server/RoutersBenchmark.java

```diff
@@ -62,7 +63,7 @@
 
     @Benchmark
     public Routed<ServiceConfig> exactMatch() {
-        final RoutingContext ctx = DefaultRoutingContext.of(HOST, "localhost", METHOD1_HEADERS.path(),
+        final RoutingContext ctx = DefaultRoutingContext.of(HOST, "localhost", UUID.randomUUID(), METHOD1_HEADERS.path(),
```

Comment:
Seems like this line is longer than 112 which we constraint using checkstyle.
Could you rebase master and do `./gradlwe checkstyle` so that you can notice the violation?

Reply:
How about making a field for `uuid` and using it here?

Reply:
+1